frontend/src/theme/primitives/spacing.ts

```diff
@@ -30,4 +30,5 @@ export const spacing = {
   xl: "1.25rem",
   twoXL: "1.5rem",
   threeXL: "2rem",
```

Comment:
Wasn't sure if I should add the new rem specification for large gap (4 rem) to the theme's spacing object... Can remove this if its unnecessary.